src/printer.js

```diff
@@ -3859,6 +3859,10 @@ function makeString(rawContent, enclosingQuote) {
   // Matches _any_ escape and unescaped quotes (both single and double).
   const regex = /\\([\s\S])|(['"])/g;
 
+  // Matches any unnecessarily escaped character.
+  // Adapted from https://github.com/eslint/eslint/blob/de0b4ad7bd820ade41b1f606008bea68683dc11a/lib/rules/no-useless-escape.js#L27
+  const regexUnnecessaryStringEscapes = /(^|[^\\])\\([^\\nrvtbfux\r\n\u2028\u2029"'0])/;
```

Comment:
Thanks! I had gotten as including the `(?:\\\\)*`, but I forgot about checking the previous character to make sure it's not a slash. I've incorporated the suggested changes as of https://github.com/prettier/prettier/pull/1575/commits/20dc94bb7d79cdb0cd6fc98c8f5dea9ff57f296e, but haven't quite gotten one test to pass... https://travis-ci.org/prettier/prettier/jobs/230873975#L1082